packages/web-components/src/components/va-process-list/va-process-list-item.tsx

```diff
@@ -60,7 +60,7 @@ export class VaProcessListItem {
           <div class="usa-process-list__heading-eyebrow">{statusTextMap[status]}</div>
           : null
         }
-        {header ? <HeaderTag class='usa-process-list__heading'>{header}</HeaderTag> : null}
+        {header ? <HeaderTag class='usa-process-list__heading'>{header}</HeaderTag> : <slot name="header"/>}
```

Comment:
This is for sure a valid approach but previously we have just add the `part` attribute for teams to style through the shadow dom. Are there any pros or cons for that approach vs a slot?

Reply:
For example:

Reply:
Good find, I think the example you found will be a better solution with far less freedom to stick an element in the header that should not be there. I've updated the PR